src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Card from "../../components/ForecastSummary";
+
+describe("Card", () => {
+  const validProps = [
+    {
+      date: 1111111,
+      description: "Stub description",
+      icon: "stubIcon",
+      temperature: {
+        min: 12,
+        max: 22,
+      },
+    },
+    {
+      date: 2222222,
+      description: "Stub description2",
+      icon: 602,
+      temperature: {
+        max: 24,
+        min: 13,
+      },
+    },
+  ];
+
+  it("renders correctly", () => {
```

Comment:
This is working, but looking at the snapshot all you can see here is the loading screen. Could an async test be added to see if the component renders properly given a bit of time to load up with the passed in data?